tests/unittests/test_ds_identify.py

```diff
@@ -113,6 +114,22 @@
 MOCK_UNAME_IS_FREEBSD = {"name": "uname", "out": UNAME_FREEBSD, "ret": 0}
 MOCK_UNAME_IS_OPENBSD = {"name": "uname", "out": UNAME_OPENBSD, "ret": 0}
 MOCK_UNAME_IS_WSL = {"name": "uname", "out": UNAME_WSL, "ret": 0}
+MOCK_WSL_INSTANCE_DATA = {
+    "name": "Noble-MLKit",
+    "distro": "ubuntu",
+    "version": "24.04",
+    "os_release": (
```

Comment:
style: We typically typically prefer `textwrap.dedent()` for more readable file strings, but I wont force you to change that - it works.